workerEngineReduce.py

```diff
@@ -1,131 +1,59 @@
-from cgitb import strong
-from email.mime import base
-from pstats import Stats
-from re import M
 import chess.svg
 import requests
 import scipy.stats as st
 import numpy as np
 import time
 import copy
 
-from config import config 
-
 import chess
 import chess.engine
 import re
 import logging
 
 
-if (config.CAREABOUTENGINE == 1):
-    engine = chess.engine.SimpleEngine.popen_uci(config.ENGINEPATH) #WHERE THE ENGINE IS ON YOUR COMPUTER
-    engine.configure({"Hash": config.ENGINEHASH})
-    engine.configure({"Threads": config.ENGINETHREADS})
-    logging.getLogger('chess.engine').setLevel(logging.INFO)
-
-#function to calculate percentage win rates for each colour, if more than 0 games
-def calc_percs(white, black, draws):

```

Comment:
I moved those 2 methods inside the class, to be able to access settings, engine, and status passed from the GUI